templates/ords-remix-jwt-sample/ords/migrateScripts/autoRESTEnableObjects.js

```diff
@@ -72,32 +72,32 @@ async function autoRESTEnableObjects(schemaName, endpoint, basicAuth) {
       objectName: 'SEARCH_VIEW',
       objectType: VIEW_OBJECT_TYPE,
       objectAlias: 'search_view',
-      isAutoRestAuth: true,
+      isAutoRestAuth: false,
     },
     {
       objectName: 'SEARCH_ARTIST_VIEW',
       objectType: VIEW_OBJECT_TYPE,
       objectAlias: 'search_artist_view',
-      isAutoRestAuth: true,
+      isAutoRestAuth: false,
     },
     {
       objectName: 'SEARCH_VENUES_VIEW',
       objectType: VIEW_OBJECT_TYPE,
       objectAlias: 'search_venues_view',
-      isAutoRestAuth: true,
+      isAutoRestAuth: false,
     },
   ];
   const autoRESTenableStatements = `
     BEGIN
-        ${objectsToRESTEnable.filter((object) => object.isAutoRestAuth).map((object) => enableObject(schemaName, object.objectName, object.objectType, object.objectAlias, object.isAutoRestAuth)).join(' ')}
+        ${objectsToRESTEnable.map((object) => enableObject(schemaName, object.objectName, object.objectType, object.objectAlias, object.isAutoRestAuth)).join(' ')}
         COMMIT;
     END;
     /
     `;
   const statementsResponse = await postRequest(endpoint, autoRESTenableStatements, basicAuth);
   printResponse(statementsResponse);
   // Add the SQL Developer role to the objects that where autoREST Enabled.
-  objectsToRESTEnable.map(async (object) => {
+  objectsToRESTEnable.filter((object) => object.isAutoRestAuth).map(async (object) => {
     const grantSQLDevStmt = grantSQLDeveloperRole(
```

Comment:
Only grant the SQL Developer role to  objects whose `isAutoRestAuth` property is equal to `true`. Add a filter operation here.